osbuild.spec

```diff
@@ -36,6 +36,8 @@ Requires:       systemd
 Requires:       tar
 Requires:       util-linux
 Requires:       python3-%{pypi_name} = %{version}-%{release}
+Requires:       python3-httpx
```

Comment:
These packages are sadly not available in both RHEL 8 and RHEL 9. I'm afraid that if we decide that want to have httpx source, we would have to ship it in a subpackage and it would be usable only in non-EL distro definitions.

Reply:
If I think very freely, it might be possible though much more invasive to have a http-source that selects either curl, or httpx based on what's installed as 'extras'. That would however change the dynamic of this PR, I'll simmer on it. Thank you.

Reply:
I think it wouldn't be so bad to have this as a subpackage and use it for fedora only at first. We can always add the dependency to RHEL if it turns out to be worth it. Fedora first and all that ;)